pkg/apis/ceph.rook.io/v1/network.go

```diff
@@ -32,10 +32,19 @@ func (n *NetworkSpec) IsMultus() bool {
 	return n.Provider == NetworkProviderMultus
 }
 
-// IsHost get whether to use host network provider. This method also preserve
-// compatibility with the old HostNetwork field.
+// IsHost can be used to determine if pods should use host networking by
+// checking whether the cephCluster is configured to use the "host" network provider.
+// This method also maintains compatibility with the old hostNetwork setting
 func (n *NetworkSpec) IsHost() bool {
-	return (n.HostNetwork && n.Provider == NetworkProviderDefault) || n.Provider == NetworkProviderHost
+	return (n.HostNetwork && n.Provider == NetworkProviderDefault) || n.Provider == NetworkProviderHost || n.Provider == NetworkProviderHostStrict
+}
+
+// IsHostStrict determines whether the cephCluster is configured to use the use host
+// or networking on all pods. This is achieved by checking whether the "host-strict" network provider
```

Comment:
I think maybe the "or" is a typo? I can't quite make sense of the phrase "configured to use host or networking".


```suggestion
// networking on all pods. This is achieved by checking whether the "host-strict" network provider
```